e2e/tests/global.setup.ts

```diff
@@ -15,6 +15,18 @@ setup("Setup solicitor user", async ({ page }) => {
 });
 
 setup("Retrieve bearer token for citizen user creation", async () => {
```

Comment:
Should this be "Retrieve bearer token for courtNav DA case creation"?

Reply:
ah need to swap my titles rounds, got them mixed up